simplefs-fuse/build.rs

```diff
@@ -4,24 +4,27 @@ extern crate pkg_config;
 use std::env;
 use std::path::PathBuf;
 
-
 pub fn main() {
-   // link to the fuse lib 
-   println!("cargo:rustc-link-lib=dylib=fuse");
+    // link to the fuse lib
```

Comment:
whoa weird what happened here!?

Reply:
This is from cargo fmt

Reply:
yeah it's really weird to me that my editor didn't complain? what does the .editorconfig file do? wonder if I got something misbehaving from it?

Reply:
It doesn’t do anything if you don’t have a plug-in.  It if you do it should line up our spacing and encodings